RockLib.Messaging/NullableAttributes.cs

```diff
@@ -0,0 +1,29 @@
+﻿#if NET48
+// This code is copied from https://source.dot.net/#System.Private.CoreLib/NullableAttributes.cs
+// and is only used for .NET 4.8
+using System;
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    /// <summary>
+    /// Specifies that an output may be null even if the corresponding type disallows it.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property | AttributeTargets.ReturnValue, Inherited = false)]
+    public sealed class MaybeNullAttribute
+        : Attribute { }
+
+    /// <summary>Specifies that when a method returns <see cref="ReturnValue"/>, the parameter may be null even if the corresponding type disallows it.</summary>
```

Comment:
Can the `<summary>` tags be placed on separate lines to stay consistent for the formatting in the rest of the project?

Reply:
Sure. FWIW That was copied verbatim from .NET's source code.